### DISCUSSION TITLE
Would you help me submit a PR contribution

### DISCUSSION THREAD

**Discussion by DsDev1** [Nixtla/mlforecast#115] [General]
When I'm trying to push my changes with following command `git push -u origin the_new_branch_name` I'm getting following error:

```
ERROR: Permission to Nixtla/mlforecast.git denied to DsDev1.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by jmoralez** [comment]
Hi. You need to:
1. Fork the repo
2. Add your fork as a remote, e.g. `git remote add upstream <your fork url>`
3. `git push -u upstream <your-branch-name>`
4. Open a PR from your fork to this repo
